src/routes/index.js

```diff
@@ -1,11 +1,18 @@
 import validator from 'validator'
 import { Router } from 'express'
+import { sign } from 'jsonwebtoken'
+import * as micromatch from 'micromatch'
 import { getDocument } from './helpers/document'
 import { transformJsonLD } from './helpers/transformers'
 import { info } from '../utils/logger'
 
 const router = new Router()
 
+const JWT_ISSUER = process.env.JWT_ISSUER || 'https://trompamusic.eu'
+const JWT_SECRET = process.env.JWT_SECRET
+const JWT_AUTH_KEYS = process.env.JWT_AUTH_KEYS ? JSON.parse(process.env.JWT_AUTH_KEYS) : []
+const JWT_EXPIRES = process.env.JWT_EXPIRES || '1d'
```

Comment:
Does this mean that tokens expire after a day? It'd be good to put this in the documentation file

Reply:
Good point I will add this the documentation.